src/test/resources/com/puppycrawl/tools/checkstyle/checks/annotation/annotationlocation/InputAnnotationLocationEnumField.java

```diff
@@ -0,0 +1,30 @@
+package com.puppycrawl.tools.checkstyle.checks.annotation.annotationlocation;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Target;
+
+/**
+ * This test-input is intended to be checked using following configuration:
+ *
+ * tokens = ["ENUM_CONSTANT_DEF", "ENUM_DEF"]
+ *
+ */
+enum InputAnnotationLocationEnumField {
```

Comment:
please add 2 annotations on enum also  (with and without violation)

Reply:
done